### ISSUE TITLE
Remove stable branch

### ISSUE THREAD

**Issue by olemb** [mido/mido#300]
Two reasons:

* it's called latest on Read the Docs (the only place I'm aware of where it's used)
* the releases are always stable
* latest (as in latest release) is more descriptive than stable

**Comment by rdoursenaud** [comment]
I don't think that this use case warrants a specific branch.
In RTD parlance, latest is an alias to the latest version of the software for which the documentation has been built. A bit like HEAD in git which is an alias to the latest commit of the currently selected branch.
IMHO the RTD build should be triggered upon adding a [semver](https://semver.org) formatted tag to the repository (vx.y.z, eg. v2.0.0).

TL;DR: I'd rather remove the stable branch entirely as it doesn't seem to be required.

**Comment by olemb** [comment]
Sounds good. Just remove the branch. (The docs have already been built.) We can reconfigure RTD when we release the new version.

**Comment by rdoursenaud** [comment]
@olemb Could you add me as a maintainer to the RTD project?
I'd like to peek into its configuration a little.
My username is the same there: https://readthedocs.org/profiles/rdoursenaud/

**Comment by rdoursenaud** [comment]
Done!
I removed the stable branch and added a 1.2.x based on the 1.2.10 tag to allow cherry-picking python 2 compatible fixes if need be.
